### PR TITLE
WebGL display in CODP

### LINKED ISSUE #845: Update event display to WebGL version
The event display in stand-alone and in the previewer should be update to use ispy-webgl.

### PR DESCRIPTION
**Pull request by tpmccauley**
ispy-webgl is now used instead of the old canvas-based display
* my fork of invenio-previewer-ispy is still included in the requirements file; once my PR for that is merged the master of invenio-previewer-ispy can be used
* closes #845

### CODE REVIEW

File: invenio_opendata/base/templates/visualise_events.html [hunk 1]
```diff
@@ -442,274 +359,7 @@
...
-}
-
-$("#switches").on("DOMNodeInserted", "tr", function(e) {
-  if ($(e.target).attr('class') === 'group') {
-    var help_name = $(this).find("td.group").html();
-    var id = "help-detsystem-" + help_items.indexOf(help_name);
-    $(this).find("td.group").append('<a href="#" onclick="openHelp(event,\''+id+'\');" class="help-detsystem"><i class="fa fa-question"></i></a>');
-  }
-});
+  console.log(ispy.version);
```
Comment (jirikuncar) [inline invenio_opendata/base/templates/visualise_events.html]: please remove debugging information

File: requirements.txt [hunk 2]
```diff
@@ -1,12 +1,12 @@
 # START
 
 # from invenio/requirements.txt
-#
 -e git+git://github.com/mrjoes/flask-admin.git#egg=Flask-Admin-1.0.9.dev0
--e git+https://github.com/inveniosoftware/invenio-previewer-ispy.git#egg=invenio-previewer-ispy
+-e git+https://github.com/tpmccauley/invenio-previewer-ispy.git@webgl#egg=invenio-previewer-ispy
```
Comment (jirikuncar) [inline requirements.txt]: it should be changed back when inveniosoftware/invenio-previewer-ispy#8 is merged

**Comment by tiborsimko** [comment]
Works as a standalone event viewer, there are some issues on individual pages.  Let's merge and deploy on DEV for further testing (and sort eventual problems afterwards).

**Comment by tiborsimko** [comment]
Thanks, squashed and merged as b45191c9bef9c93f6e82f25ab23b310e1949b128.

**Comment by tiborsimko** [comment]
FYI deployed on DEV, everything's working in both standalone and embedded modes:

- http://opendatadev.cern.ch/visualise/events/CMS
- http://opendatadev.cern.ch/record/600

**Pull request closed without merging**